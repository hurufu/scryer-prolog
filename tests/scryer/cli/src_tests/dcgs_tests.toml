args = [
    "-f",
    "--no-add-history",
    "-g", "use_module(library(dcgs_tests))",
    "-g", "use_module(library(dcgs))",
    "-g", "use_module(library(dif))",
    "-g", "use_module(library(lambda))",
    "-g", "use_module(library(lists))",
    "-g", "dcgs_tests:main(dcgs_tests)",
    "src/tests/dcgs.pl"
]
