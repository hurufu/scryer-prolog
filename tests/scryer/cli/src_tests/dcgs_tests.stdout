Running test "cuts inside DCG do not bleed to outer scope #1"
Running test "cuts inside DCG do not bleed to outer scope #2"
Running test "cuts inside DCG do not bleed to outer scope #3"
Running test "cuts inside DCG do not bleed to outer scope #4"
Running test "cuts inside DCG do not bleed to outer scope #5"
Running test "cuts inside DCG do not bleed to outer scope #6"
Running test "cuts inside DCG do not bleed to outer scope #7"
Running test "cuts inside DCG do not bleed to outer scope #8"
